build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -21,11 +21,22 @@ const jsModuleErrorsExtension = '.ddc.js.errors';
 const jsModuleExtension = '.ddc.js';
 const jsSourceMapExtension = '.ddc.js.map';
 const metadataExtension = '.ddc.js.metadata';
+const fullKernelExtension = '.ddc.full.dill';
 
 /// A builder which can output ddc modules!
 class DevCompilerBuilder implements Builder {
   final bool useIncrementalCompiler;
 
+  /// Make ddc generate full dill for libraries it compiles.
+  ///
+  /// Full dill is only generated in debug mode and does not include any
```

Comment:
This comment about "debug mode" feels misplaced. At best we can say "Full dill is only generated by default in debug mode," but I don't think even that is true, since we haven't updated the `build.yaml` file to set this option by default.

Reply:
the comment is no longer true, I will remove it

Reply:
I got confused by debug mode - apparently debug mode settings is always on in my case since ddc is only used in debug mode. I will remove the comment.

Reply:
fixed